solutions/fully-configurable/scripts/delete_secrets.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+API_KEY=$2
+
+iam_cloud_endpoint="${IBMCLOUD_IAM_API_ENDPOINT:-"iam.cloud.ibm.com"}"
+IBMCLOUD_IAM_API_ENDPOINT=${iam_cloud_endpoint#https://}
+
+iam_token=$(curl -s -X POST "https://${IBMCLOUD_IAM_API_ENDPOINT}/identity/token" --header 'Content-Type: application/x-www-form-urlencoded' --header 'Accept: application/json' --data-urlencode "grant_type=urn:ibm:params:oauth:grant-type:apikey" --data-urlencode "apikey=$API_KEY" | jq -r '.access_token')  # pragma: allowlist secret
+
+secret_group_id=$1
+secrets_manager_crn=$3
+secrets_manager_endpoint=$4
+
+secrets_manager_instance_id=$(echo "$secrets_manager_crn" | cut -d ':' -f8)
+secrets_manager_region=$(echo "$secrets_manager_crn" | cut -d ':' -f6)
```

Comment:
how about changing the script to accept `secrets_manager_instance_id` and `secrets_manager_region` as arguments instead of asking for the `secrets_manager_crn`. That way you don't have to use some crazy `cut` logic to parse the info